source/MenuItem.hpp

```diff
@@ -57,17 +57,17 @@ namespace GTA
 		virtual void Change(bool right);
 		virtual void SetOriginAndSize(System::Drawing::Point topLeftOrigin, System::Drawing::Size size);
 
-		void UpdateText();
+		virtual void UpdateText();
 
 		virtual property MenuBase ^Parent;
 		virtual property System::String ^Caption;
 		virtual property System::String ^Description;
+		property UIText ^mText;
```

Comment:
Any reason for this property? Or did you implement it so there is more freedom for devs?